cmd/helm/list.go

```diff
@@ -95,8 +104,60 @@ func newListCmd(cfg *action.Configuration, out io.Writer) *cobra.Command {
 	f.BoolVar(&client.Pending, "pending", false, "show pending releases")
 	f.BoolVar(&client.AllNamespaces, "all-namespaces", false, "list releases across all namespaces")
 	f.IntVarP(&client.Limit, "max", "m", 256, "maximum number of releases to fetch")
-	f.IntVarP(&client.Offset, "offset", "o", 0, "next release name in the list, used to offset from start value")
```

Comment:
`o` was already being used on all other commands as the short hand for output, if we think that removing this shorthand is going to be too big of a change, let me know